lte/gateway/c/oai/tasks/nas/emm/Attach.c

```diff
@@ -710,6 +710,15 @@ int emm_proc_attach_complete(
           (nas_emm_attach_proc_t*)
               ue_mm_context->emm_context.emm_procedures->emm_specific_proc;
 
+      /* Process attach complete msg only if T3450 timer is running
+       * If its not running it means that implicit detach is in progress
+       */
+      if (attach_proc->T3450.id == NAS_TIMER_INACTIVE_ID) {
+        OAILOG_WARNING(
```

Comment:
nit: can we use `OAILOG_WARNING_UE` to prepend the IMSI here? would help for tracing certain subscribers

Reply:
second that...